docs/nightly/en/user-guide/logs/quick-start.md

```diff
@@ -1,186 +1,234 @@
++---------+----------------------+------+------+---------+---------------+
+| message | String               |      | YES  |         | FIELD         |
+| time    | TimestampMillisecond | PRI  | NO   |         | TIMESTAMP     |
++---------+----------------------+------+------+---------+---------------+
 ```
 
-The query result is as follows:
+```sql
+DESC pipeline_logs;
+```
 
 ```sql
- Column |        Type         | Key | Null | Default | Semantic Type
---------+---------------------+-----+------+---------+---------------
- id1    | Int32               |     | YES  |         | FIELD
- id2    | Int32               |     | YES  |         | FIELD
- type   | String              | PRI | YES  |         | TAG
- logger | String              | PRI | YES  |         | TAG
- log    | String              |     | YES  |         | FIELD
- time   | TimestampNanosecond | PRI | NO   |         | TIMESTAMP
-(6 rows)
++---------------+---------------------+------+------+---------+---------------+
+| Column        | Type                | Key  | Null | Default | Semantic Type |
++---------------+---------------------+------+------+---------+---------------+
+| ip_address    | String              | PRI  | YES  |         | TAG           |
+| http_method   | String              | PRI  | YES  |         | TAG           |
+| status_code   | String              | PRI  | YES  |         | TAG           |
+| request_line  | String              |      | YES  |         | FIELD         |
+| user_agent    | String              |      | YES  |         | FIELD         |
+| response_size | Int32               |      | YES  |         | FIELD         |
+| timestamp     | TimestampNanosecond | PRI  | NO   |         | TIMESTAMP     |
++---------------+---------------------+------+------+---------+---------------+
+7 rows in set (0.01 sec)
 ```
 
-From the above result, we can see that based on the processed result of the pipeline, the `public.logs` table contains 6 fields: `id1` and `id2` are converted to the `Int32` type, `type`, `log`, and `logger` are converted to the `String` type, and time is converted to a `TimestampNanosecond` type and indexed as Timestamp.
+From the table structure, you can see that the `origin_logs` table has only two columns,
+with the entire log message stored in a single column.
+The `pipeline_logs` table stores the log message in multiple columns.
+
+It is recommended to use the pipeline method to write logs to GreptimeDB.
+Here are some advantages of using the pipeline method to split the log message into multiple columns:
+
+- Smaller data storage space: The tag index size is less than the full-text index size; fewer full-text data leads to a smaller full-text index size; Tag-structured data have high compaction rates.
+- High performance: Querying tags is more efficient than querying full-text data.
+- Writing SQL is easier: You do not need to remember many full-text query syntaxes, just search by tags.
```

Comment:
Splitting a whole string message into different decisive columns itself gives the advantage of explicitly querying a certain value within one certain column, which is already better than fuzzy query upon strings. Marking it as Tag creates an inverted index upon the column values, which speeds up the query.

So even searching Field columns should be way faster since it's exact matching.